src/components/application_manager/include/application_manager/resumption/resumption_data_json.h

```diff
@@ -49,9 +49,15 @@ class ResumptionDataJson : public ResumptionData {
    * @brief Constructor of ResumptionDataJson
    */
   ResumptionDataJson(
-      LastState& last_state,
+      resumption::LastStateWrapperPtr last_state_wrapper,
       const application_manager::ApplicationManager& application_manager);
 
+  DEPRECATED
+  ResumptionDataJson(
```

Comment:
@LitvinenkoIra Do we still store DEPRECATED code? I though that we got rid of it.